presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -39,41 +42,56 @@ public class PlanNodeStats
 
     private final Duration planNodeScheduledTime;
     private final Duration planNodeCpuTime;
+    private final Duration planNodeBlockedWallTime;
```

Comment:
why are these added here?

Reply:
The main driver behind this change is that we have an external tool that helps visualize some plans from the JSON output of the event listener. The event listener output is the actually the same output as what you get from an `EXPLAIN ANALYZE`. However, you only get execution statistics in the text output from the event listener, but not the JSON one. So, to augment the event listener's JSON plan output to be closer to the text version, I added some more statistics to this class.